### PR TITLE
Update quickstart.md

### PR DESCRIPTION
**Pull request by gregersen12**
Missing the part with changing URI for omniauth

### CODE REVIEW

**Comment by julionc** [comment]
:+1: maybe, we can use env variable too.

File: QUICKSTART.md [hunk 1]
```diff
@@ -61,13 +61,20 @@ git add .
...
 
-6. Deploy
+6. Changing redirect URI in Omniauth
+------------------------------------
+```
+Go to your omniauth config file found at /config/initializers/omniauth and make sure to set the redirect URI.
+Set it to your APPNAME.herokuapp.com/auth/shopify/callback adress, as it will default to localhost:3000 on app creation.
+```
+
+7. Deploy
```
Comment (kevinhughes27) [inline QUICKSTART.md]: this shouldn't be required - the redirect_uri is set by the Shopify App Initializer using the parameter passed in the previous step.

**Comment by kevinhughes27** [comment]
This example doesn't use the ENV since its an added step. Its pretty clear in the main README that we recommend using ENV for production apps. If you think its worth adding a note here too I am okay with that.

**Comment by gregersen12** [comment]
That's true @kevinhughes27. I realized, however, that the shopify_app gem per default sets the URI to localhost:3000 instead of the url in which the app lives. Now this is rather obvious, but for someone undergoing first-time setup the app will not work untill this is set correctly.

I pointed at the wrong file by the way. I ment to edit the shopify_app initializer.

**Comment by kevinhughes27** [comment]
How about we add a note to step 6 explaining it a bit rather than adding a new step?

**Comment by gregersen12** [comment]
Seems reasonable Kevin :-)

**Comment by kevinhughes27** [comment]
cool! let me know when you've updated and I'll take a look

**Comment by gregersen12** [comment]
Forgive me for being unaware, but how do I edit my pull-request / update the 'code'?

**Comment by kevinhughes27** [comment]
If you push new commits or amend your current commit (preferred) and push it back to github the pull request will update

**Comment by gregersen12** [comment]
I have updated the code, and I realize I wanted to edit this because I was a toon and didn't realize that the redirect-uri is added on creation :P

File: QUICKSTART.md [hunk 2]
```diff
@@ -59,6 +59,9 @@ use the keys from your app in the partners area
 rails generate shopify_app -api_key=a366cbafaccebd2f615aebdfc932fa1c -secret=8750306a895b3dbc7f4136c2ae2ea293 -redirect_uri=https://<name>.herokuapp.com/auth/shopify/callback
 git add .
 git commit -m 'generated shopify app'
+
+If you forget to set your keys or redirect uri above you will find them in the shopify_app initializer at: /config/initializers/.
+In light of this section we recommend adding a gem or utilizing ENV variables to handle your keys.
```
Comment (kevinhughes27) [inline QUICKSTART.md]: nitpick but can you just add `handle your keys before releasing your app.`

**Comment by kevinhughes27** [comment]
small thing I think we should add. This is a nice improvement!

**Comment by kevinhughes27** [comment]
woops this never got merged - can you rebase and I'll get it in!

**Pull request closed without merging**